src/ParseRevisions.py

```diff
@@ -92,7 +92,7 @@ def parseItem(line):
 
     arr = line.split()
     # remove unwanted text in description
-    if arr[0] == "v" or arr[0] == "Rev." or arr[0] == "Version":
+    if len(arr) > 0 and (arr[0] == "v" or arr[0] == "Rev." or arr[0] == "Version"):
```

Comment:
Similar to the issue in content parsing, added length check to avoid crashes on unexpectedly formatted files